docs/contribute/code.md

```diff
@@ -40,6 +40,14 @@ If everything is working as it should, you should see the `example/cra-ts-essent
 
 ![Example Storybook running](./storybook-cra-examples-optimized.png)
 
+### End-to-end tests (e2e)
+
+In order to test key parts of storybook functionality, end-to-end tests using [cypress](https://www.cypress.io/) are run during CI. If you would like to also run these tests on your own machine, there are a few steps to take:
+
+1. Make sure that you have built storybook on your branch with `yarn bootstrap --core`.
+2. Publish storybook packages to a local registry in one terminal: `yarn local-registry --port 6000 --open --publish`
+3. In a second terminal, create a repro against the local registry and run cypress against it: `yarn test:e2e-framework`.
```

Comment:
@IanVS I've been checking this and we have a bit of room for improvement, we could move this below and place it above the "Submit a pull request" section of the docs and frame it as follows:

```md
### End-to-end tests (e2e)

Storybook's monorepo is set up to rely on end-to-end testing with [Cypress](https://www.cypress.io/) during CI. To help with testing, we encourage running this test suite before submitting your contribution. Detailed below are some steps you can take:

1.  Open a terminal and run `yarn local-registry --port 6000 --open --publish` to publish Storybook's packages into a local registry.
2.  In a second terminal, set up a reproduction using the local registry and run the Cypress tests with `yarn test:e2e-framework`.
```

The rationale behind this change is based on the following. With your instructions, we're teaching the contributor to run the tests while, or after making his/her change. As were it was placed before it was placed a bit awkward as he/she wouldn't have published Storybook's packages or began working.

Would love to hear your thoughts on this.

Reply:
Sure that makes sense!  I'm not so sure about removing the step instructing to build the project, though, since that has bitten me _personally_ in the past, and it's always frustrating to realize I forgot to build the project, then need to go figure out _how_.

Reply:
Sounds reasonable, I was looking at it from a documentation flow, but we can introduce it back and frame something like:
```md
1. Ensure you have Storybook successfully built in your local branch (i.e., run `yarn bootstrap --core`)
2. ...
3. ....
```

Sounds good?

Reply:
Yeah, often I don't read the whole documentation page, and just find the section I'm looking for.  :shame:

Reply:
@IanVS same here,  it has happened to me more times than I can count as well 🙂!